### DISCUSSION TITLE
Error: The release is not complete and cannot be loaded. main.js is missing from the release

### DISCUSSION THREAD

**Discussion by JeffreyQin** [TfTHacker/obsidian42-brat#60] [Q&A] [answered]
I just installed the Brat plugin to test the beta version of a plugin I made for my company. When I was using the command "add a plugin for beta testing", I received the following error

"The release is not complete and cannot be loaded. main.js is missing from the release."

However, I have ensured that my release is indeed complete and that main.js is not missing (shown in screenshot below). Are there other possible causes for this issue case?

<img width="2190" alt="image" src="https://github.com/TfTHacker/obsidian42-brat/assets/122770444/219369e5-6f67-431c-9c89-0e2ac602cc36">

**Comment by TfTHacker** [comment]
For which plugin is this?

Just a reminder, on the root folder of a repo there needs to be a manifest file and the manifest file contains a version number to a "release". The release has to exists.

BRAT models the entire installation requirements of Obsidian. So if something isn't working, it is a good sign it won't work with Obsidian either. (the value of testing :-) )

**Reply by JeffreyQin** [reply]
Thank u for the reply. This is for a company plugin I made. The source code is found here
https://github.com/enst/genvoiceObsidian

I have ensure that all the package.json or manifest.json are showing version 1.0.7, with the release being so too. In addition, it has been running fine on Obsidian, but I just wish to use this plugin for better update. It will be nice if u can help take a look at the code and see if there are any issues with the versioning :)

**Comment by TfTHacker** [comment, marked as answer]
Hi, Everything looks great, except a tiny detail. Obsidian requires that the release is "Tagged" with the version number, and so Brat is looking for this. You named your release with the version number, but it needs to be tagged with the version number.

Here is an example of how it looks when it's tagged:

![image](https://github.com/TfTHacker/obsidian42-brat/assets/69121180/4807f8be-8992-4184-bd9a-6060106eea77)

**Reply by JeffreyQin** [reply]
I changed the tag and it worked perfectly fine. I tried the update function too and it has made everything so much easier.

Thank you so much for your time. Beautiful plugin.

**Comment by TfTHacker** [comment]
It was my pleasure, and I wish you success in your work. Also, kudos to the "testers" spirit and good quality software.
